filters/openpolicyagent/internal/util/util.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+
+	"github.com/zalando/skipper/filters"
+)
+
+func Uuid4() (string, error) {
+	bs := make([]byte, 16)
+	_, err := io.ReadFull(rand.Reader, bs)
+	if err != nil {
+		return "", err
+	}
+	bs[8] = bs[8]&^0xc0 | 0x80
+	bs[6] = bs[6]&^0xf0 | 0x40
+	return fmt.Sprintf("%x-%x-%x-%x-%x", bs[0:4], bs[4:6], bs[6:8], bs[8:10], bs[10:]), nil
+}
```

Comment:
Do we need this exported? Does it have to be an uuid?
I think we also have uuid library indirectly imported.
It would also be random on each call for the same config

Maybe we could use content id like:
```go
opaConfig, err := config.ParseConfig(configBytes, fmt.Sprintf("%x", sha256.Sum256(configBytes)))
```

Reply:
I'll change this to the library. It needs to be random even for the same config (the id references the OPA instance that can be identified from the control plane)